### ISSUE TITLE
Update translations for 0.5.0

### ISSUE THREAD

**Issue by rafaelmardojai** [rafaelmardojai/blanket#166]
`0.5.0` will be released soon, if you have time to update the translations would be very awesome.

Thanks in advance!

- [ ] Catalan @fitojb @juxuanu 
- [ ] Czech  @pervoj 
- [ ] German @eladyn 
- [ ] Esperanto @Porrumentzio 
- [ ] Spanish  @rafaelmardojai 
- [ ] Basque @Porrumentzio 
- [ ] Persian  @mskf1383 
- [ ] Finnish @heidiwenger 
- [ ] French  @maoschanz @julroy67 
- [ ] Irish @Moilleadoir 
- [ ] Galician @frandieguez 
- [ ] Croatian @milotype
- [ ] Hungarian  @osiixy 
- [ ] Indonesian @cho2 
- [ ] Italian @albanobattistella 
- [ ] Japanese
- [ ] Nepali @IBA4 
- [ ] Dutch @Vistaus 
- [ ] Occitan @Mejans 
- [ ] Portuguese @JulianoSCbah @hugok79 
- [ ] Brazilian Portuguese @pedrofleck 
- [ ] Russian @LeftyDeMilo 
- [ ] Slovak @MartinIIOT 
- [ ] Swedish @eson57 
- [ ] Tamil @123survesh 
- [ ] Turkish @caglarturali 
- [ ] Ukrainian @IhorHordiichuk 
- [ ] Simplified Chinese @apocelipes

**Comment by haggen88** [comment]
@rafaelmardojai  I think the translation of Boat in Spanish should be "Bote" no "Barco"

**Comment by rafaelmardojai** [comment]
> @rafaelmardojai I think the translation of Boat in Spanish should be "Bote" no "Barco"

Yeah, makes sense, I don't remember why I choose "Barco", maybe the sound gave me the impression of a big boat :P. I will fix that.

**Comment by eladyn** [comment]
I probably won't be able to update the translations within the next week or so, but maybe I can finish them in time so that they can be part of the next release.

**Comment by hugok79** [comment]
> > @rafaelmardojai I think the translation of Boat in Spanish should be "Bote" no "Barco"
> 
> Yeah, makes sense, I don't remember why I choose "Barco", maybe the sound gave me the impression of a big boat :P. I will fix that.

“Barco” is Portuguese for “Boat”

**Comment by juxuanu** [comment]
In Spain, people say Barco and not Bote. Bote here usually refers to an smaller, auxiliary boat.

**Comment by haggen88** [comment]
The sound [wobble and sound of wood] refers to a small boat : https://freesound.org/people/Falcet/sounds/439368/

**Comment by hugok79** [comment]
> 
> 
> In Spain, people say Barco and not Bote. Bote here usually refers to an smaller, auxiliary boat.

It is the same in Portugal, **Boat=Barco** and **Dinghy=Bote**

**Comment by milotype** [comment]
@eladyn If you want, you could review my German update at #178

**Comment by rafaelmardojai** [comment]
@haggen88 I think I will stay with "Barco", culturally I can't think on a "Bote" with moving ropes. I would call it "Barco pequeño" but I don't think is needed :P

**Comment by haggen88** [comment]
@rafaelmardojai I leave it in your hands ... the important thing is the sounds as a concept.

**Comment by rafaelmardojai** [comment]
I will release `0.5.0` on September 28. But don't worry if you don't get to update your translation, I will make a point release when the translations left are done.

**Comment by Porrumentzio** [comment]
I don't have my usual developing environment usable and thus I can't make the translation. Did you think about using Weblate to host Blanket's translations? It would be more easier for translators (I could help on this)

**Comment by Vistaus** [comment]
> I don't have my usual developing environment usable and thus I can't make the translation. Did you think about using Weblate to host Blanket's translations? It would be more easier for translators (I could help on this)

You don't need a development environment, you just need an app like Poedit. And even if you don't have that, you can edit po files in your favorite text editor.

That being said: Weblate or a similar service would be a good addition.

**Comment by rafaelmardojai** [comment]
Yeah, I plan to use Weblate in the future (I'm already using it in other projects).
